composer.json

```diff
@@ -36,6 +36,7 @@
     "gedmo/doctrine-extensions": "^2.4"
   },
   "autoload": {
+    "files": ["src/helpers.php"],
```

Comment:
@patrickbrouwers there's no other way to load functions, right? I couldn't find documentation of namespaced functions in composer docs.
Oh, this probably breaks in php 5.5, so we should bump anyway.